src/main/resources/public/ts/model/rbs/rbs-emitter.model.ts

```diff
@@ -61,10 +61,7 @@ export class RbsEmitter {
         if (rbsSniplet && rbsSniplet.vm) {
             let bookingsContent = rbsSniplet.vm.prepareBookingsToSave();
             if (bookingsContent.length > 0) {
-                calendarEvent.bookings = new Bookings();
-                calendarEvent.bookings = {
-                    all : [...bookingsContent]
-                };
+                calendarEvent.bookings = [...bookingsContent];
```

Comment:
booking field from calendarEvent has been changed to fit both saved bookings and not saved bookings cases.